homework_5/HW5_2_country.py

```diff
@@ -0,0 +1,43 @@
+import xml.etree.ElementTree as ET
+filename = 'mondial-3.0.xml'
+path  = 'C:/Users/roman/PycharmProjects/My_project'
```

Comment:
These variables aren't used in the program.

The **parse_and_remove** function takes 'mondial-3.0.xml' value as filename and 'country' as path on line 27.